tsconfig.eslint.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "./tsconfig.json",
+    "include": [
+        "src/**/*"
+    ],
+    "exclude": [
+        "dist/**/*"
+    ]
```

Comment:
I think this is redundant (as far as I'm aware), as it is not included in the "include" anyway. I guess the whole file is redundant, as the include property specifies the same value as the extended file does.